tree/dataframe/inc/ROOT/RDF/RAction.hxx

```diff
@@ -43,7 +43,7 @@ class RActionCRTP {
 
 /// A type-erasing wrapper around RColumnValue.
 /// Used to reduce compile time by avoiding instantiation of very large tuples and/or (std::get<N>...) fold expressions.
-class RTypeErasedColumnValue {
+class R__CLING_PTRCHECK(off) RTypeErasedColumnValue {
```

Comment:
This is a speed improvement in cling, I assume?

Reply:
correct, runtime nullptr dereference checks are a performance killer and at some point we saw that `RColumnValue` contributed most of the load, so now I'm adding the annotation to all column value readers.